src/ucp/wireup/select.c

```diff
@@ -1630,8 +1631,13 @@ ucp_wireup_add_bw_lanes(const ucp_wireup_select_params_t *select_params,
         if (num_paths_override &&
             ((skip_dev_index != UCP_NULL_RESOURCE) && /* clang sanitizer */
              (skip_dev_index == dev_index))) {
+            /* Allow path since we skipped one */
             local_num_paths++;
             remote_num_paths++;
+
+            /* Remove that skipped path from score computation */
+            dev_count.local_skip[dev_index]      = 1;
+            dev_count.remote_skip[ae->dev_index] = 1;
```

Comment:
can't we do not increase dev_counts for skipped lane instead?

Reply:
ok

Reply:
maybe rename local_skip to first_path_index_local (and remote, accordingly)